src/NI-SCOPE Time Domain/README.md

```diff
@@ -0,0 +1,30 @@
+## NI-SCOPE Time Domain Measurement 
+
+This example performs time domain measurement using NI SCOPE.
+
+
+### Features
+
+- Uses the NI-SCOPE LabVIEW API.
+- Pin-aware, supporting one session and one pin.
+  - Uses the same selected measurement function and range for all selected pin/site combinations.
+- Includes InstrumentStudio project files.
+- Includes a TestStand sequence showing how to configure the pin map, register.
+  instrument sessions with the session management service, and run a measurement.
+  - For the sake of simplicity, the TestStand sequence handles pin map and session registration and unregistration in the `Setup` and `Cleanup` sections of the main sequence. For **Test UUTs** and batch process model use cases, these steps should be moved to the `ProcessSetup` and `ProcessCleanup` callbacks.
```

Comment:
Test UUTs is not a process model. This should probably say: For **Sequential** and **Batch** process models, ...